template/_package.json

```diff
@@ -80,10 +80,14 @@
     "babel-jest": "^24.1.0",
     "jest": "^24.1.0",
     "vue-jest": "^3.0.3"<% } %><% if (test === 'ava') { %>,
+    "@babel/preset-env": "^7.4.4",
```

Comment:
By default, `ava` is using @ava/stage-4 preset, do we need `@babel/preset-env` and `@babel/core` ?